components/src/GitHubCommitPicker.js

```diff
@@ -6,11 +6,28 @@ import PropTypes from 'prop-types'
 import { Typeahead, Highlighter } from 'react-bootstrap-typeahead'
```

Comment:
I'm not so sure if is fair enough to force a developer to use a specific library for styling.
If we want to release some stand-alone components, perhaps it could be better to give to the user the basic functionality without any styling implementation, and neither third-party libraries if not necessary.

Stand-alone components should be clean as much as possible, then each developer can improve the styling as he prefers.
@jeffmcaffer thoughts?

Reply:
It's a good point. We could allow folks to plug in a component to use for various cases. If that is relatively straight forward, sure. We could also do this based on demand. If people want different styling then we can adapt. I propose that we factor out a representative set of components and get a sense of the overall styling needs and build up a more holistic approach.